cluster-api/src/main/java/io/aiven/klaw/clusterapi/models/confluentcloud/Metadata.java

```diff
@@ -0,0 +1,7 @@
+package io.aiven.klaw.clusterapi.models.confluentcloud;
+
+public class Metadata {
+  public String self;
+  public Object next;
```

Comment:
Is this a known object we could model or a bit of a hot mess of things?

Reply:
All the objects defined within confluentcloud dir are defined based on their implementation